server/controllers/connectedAccounts.js

```diff
@@ -47,7 +49,24 @@ module.exports = (app) => {
       } else {
         return next(new errors.BadRequest('Github authorization failed'));
       }
-    }
+    },
 
+    fetchAllRepositories: (req, res, next) => {
+      const payload = req.jwtPayload;
+      ConnectedAccount
+      .findOne({where: {id: payload.connectedAccountId}})
+      .then(ca => {
+        const options = {
+          url: `https://api.github.com/user/repos?per_page=1000&sort=stars&access_token=${ca.secret}&type=all`,
+          headers: {
+            'User-Agent': 'OpenCollective'
+          },
+          json: true
+        };
+        return Promise.promisify(request, {multiArgs: true})(options).then(args => args[1]);
```

Comment:
Well I was just guessing that as `request` returns two values, `response` and `body`, by using `multiArgs: true` you're asking bluebird to return these into an `args` array in the same order, so in that array, `response` would be at index 0 and `body` at index 1.